data/mods/MMA/martialarts.json

```
@@ -1,4 +1,204 @@
+        "description": "You move slightly faster than normal due to a combination of confidence, raining, and clarity of mind.  This slight edge adds up with each action.\n\n-10% move cost",
+        "skill_requirements": [ { "name": "melee", "level": 3 } ],
+        "melee_allowed": true,
+        "mult_bonuses": [ { "stat": "movecost", "scale": 0.85 } ]
+      }
+    ],
+    "ondodge_buffs": [
+      {
+        "id": "mma_buff_diamond_mind_ondodge",
+        "name": "Pearl of Black Doubt",
+        "description": "With every miss, your opponents become more uncertain, their doubt growing like an irritating pearl in the mouth of a helpless oyster.\n\n+1 Dodge attempt\nLasts 1 turn.  Stacks 2 times",
+        "melee_allowed": true,
+        "buff_duration": 1,
+        "max_stacks": 2,
+        "bonus_dodges": 1
+      }
+    ],
+    "ongethit_buffs": [
+      {
+        "id": "mma_buff_diamond_mind_ongethit",
+        "name": "Mind over Body",
+        "description": "Your training and mental toughness allow you to use your focus to overcome physical threats.\n\n+1 Accuracy.\nLasts 1 turn.  Stacks 2 times",
+        "skill_requirements": [ { "name": "melee", "level": 2 } ],
+        "melee_allowed": true,
+        "buff_duration": 1,
+        "max_stacks": 2,
+        "flat_bonuses": [ { "stat": "hit", "scale": 1.0 } ]
+      }
+    ],
+    "onpause_buffs": [
+      {
+        "id": "mma_buff_diamond_mind_onpause",
+        "name": "Quicksilver Motion",
+        "description": "In the blink of an eye, you make your move.  Your speed, reflexes, and boundless confidence combine to allow you to make a fast, bold move that catches your foes off guard.\n\n+50 Speed.\nLasts 1 turn.",
+        "skill_requirements": [ { "name": "melee", "level": 5 } ],
+        "melee_allowed": true,
+        "buff_duration": 1,
+        "flat_bonuses": [ { "stat": "speed", "scale": 50.0 } ]
+      }
+    ],
+    "techniques": [ "mma_tec_diamond_mind_strike", "mma_tec_diamond_mind_crit" ],
+    "weapon_category": [ "DUELING_SWORDS", "1H_SWORDS" ]
+  },
+  {
+    "type": "martial_art",
+    "id": "style_mma_hylian",
+    "name": { "str": "Hylian Swordsmanship" },
+    "description": "This rare form of combat has been practiced by many legendary heroes throughout the ages.  Hylian Swordsmanship favors mobility for offense and defense by using spins, jumps, and flips to confuse enemies and strike from unexpected angles.",
+    "initiate": [ "You begin to step lightly from side to side.", "%s begins to step lightly from side to side." ],
+    "learn_difficulty": 10,
+    "primary_skill": "cutting",
+    "strictly_melee": true,
+    "static_buffs": [
+      {
+        "id": "mma_buff_hylian_static1",
+        "name": "Combat Acrobat",
+        "description": "Always stay light on your feet.  It is better to evade than be hit.\n\n+1.0 Dodging skill.",
+        "melee_allowed": true,
+        "flat_bonuses": [ { "stat": "hit", "scale": 1.0 } ]
```

Since it says dodging bonus and is part of a chain for dodge buffs increasing with skill...
```suggestion
        "flat_bonuses": [ { "stat": "dodge", "scale": 1.0 } ]
```